markdown/events/2021/bytesize-25-nf-core-ampliseq.md

```diff
@@ -27,3 +27,72 @@ It is our hope that these talks / videos will build an archive of training mater
+
+[6:13](https://youtu.be/a0VOEeAvETs&t=373)
+First thing I did is to compare those kind of methods. What you can see here on the y-axis is the relative abundance of sequences in a reference data set. The reference data set consists of 27 bacterial strains, which has 35 expected amplicon sequences because bacterial genomes can have several copies of that gene. Here you can see the reference, meaning this is how it should look like. There are high abundant sequences, but also very low abundant sequences. The color of the dots show if the sequence is a perfect match in green to the reference, in blue one off, so very similar, or in black cross a sequence that is not expected. You can see here that the OTU-based methods, here four columns for four different methods or settings, have quite a lot of those unexpected sequences. They also, in this region here, do not find all the ones that we would expect, the green dots here. The ASV-based methods, here two different methods, do show a much reduced number of false positives. However, some of those sequences are also not detected for this method here, that is very strict.
+
+[7:50](https://youtu.be/a0VOEeAvETs&t=470)
+I did that not only with one sample, but with three different samples and with here the OTU-based methods and the ASV-based methods, not only for precision and sensitivity of finding the original sample sequences again, but also for the taxonomic classification and for a diversity index, the Shannon diversity. Here the green color shows the best methods and the red ones, the worst ones. You can see that the ASV-based methods perform much better in the amplicon sequencing analyses than the OTU-based methods.
+
+[8:31](https://youtu.be/a0VOEeAvETs&t=511)
+Coming from that, I wanted to produce a pipeline that is doing exactly what I need and to use that kind of knowledge that I have accumulated by the benchmarking. I wanted to produce a pipeline to analyze all kinds of amplicon data with a wide range of input types, reference taxonomies, and downstream analyses. We came up in 2018 then with the nf-core/ampliseq pipeline, in the end of 2018. We released version 1.00 and did some feature and maintenance updates. We also then published a paper about it and continued afterwards in 2021 quite a lot with adding more features to the pipeline. Now it is at a pretty good stage how I imagined it in the beginning.
```

Comment:
`version 1.0.0` instead of `version 1.00`
```suggestion
Coming from that, I wanted to produce a pipeline that is doing exactly what I need and to use that kind of knowledge that I have accumulated by the benchmarking. I wanted to produce a pipeline to analyze all kinds of amplicon data with a wide range of input types, reference taxonomies, and downstream analyses. We came up in 2018 then with the nf-core/ampliseq pipeline, in the end of 2018. We released version 1.0.0 and did some feature and maintenance updates. We also then published a paper about it and continued afterwards in 2021 quite a lot with adding more features to the pipeline. Now it is at a pretty good stage how I imagined it in the beginning.
```